R/nest.R

```diff
@@ -33,10 +33,15 @@
 nest <- function(data, ..., .key = "data") {
   UseMethod("nest")
 }
+
 #' @export
 nest.data.frame <- function(data, ..., .key = "data") {
   key_var <- as_string(ensym2(.key))
 
+  if (!inherits(data, "tbl_df")) {
```

Comment:
I was thinking that we'd do this unconditionally, simply because nested data frames are so painful to work with if you don't have the nice tlbble print methods.

Reply:
Ok, done! Update committed and pushed. Thanks!